admin/users/views.py

```diff
@@ -132,47 +124,30 @@ def get_object(self, queryset=None):
 class ResetPasswordView(OSFAdmin, FormView):
     form_class = EmailResetForm
     template_name = 'users/reset.html'
-
-    def get(self, request, *args, **kwargs):
-        try:
-            return super(ResetPasswordView, self).get(request, *args, **kwargs)
-        except AttributeError:
-            return page_not_found(
-                request,
-                AttributeError(
-                    'User with id "{}" not found.'.format(
-                        self.kwargs.get('guid')
-                    )
-                )
-            )
-
-    def post(self, request, *args, **kwargs):
-        try:
-            return super(ResetPasswordView, self).post(request, *args, **kwargs)
-        except AttributeError:
-            return page_not_found(
-                request,
-                AttributeError(
-                    'User with id "{}" not found.'.format(
-                        self.kwargs.get('guid')
-                    )
-                )
-            )
+    context_object_name = 'user'
 
     def get_context_data(self, **kwargs):
         user = User.load(self.kwargs.get('guid'))
         try:
             self.initial.setdefault('emails', [(r, r) for r in user.emails])
         except AttributeError:
```

Comment:
>It is a double check that the user found with an email is the same one found with the guid.

"It" being `form_valid`, I take it, as that's not what this is doing. Also, should `form_valid` then be raising something other than a `404`, then, as it might not be the case that an object wasn't found? `400` or `409` both seem more correct.